### ISSUE TITLE
Post Migration - ordering of episode messed up

### ISSUE THREAD

**Issue by ksri99** [albertobeta/PodcastGenerator#59]
Hello,

Thanks for this wonderful tool.  I have been using this for publishing some non-profit activity related podcasts.  We had to move to a new server and the entire file system from old server has been copied over to the new server.  As the path changed, the config.php has been updated to reflect new path.  The page loads and the podcast works, but the issue is, since the file timestamps have been updated (due to copy), all the podcasts now show out of order.  They lost the chronological setup that we did.  I have 160+ episodes so updating each would be really painful.  Am I doing something wrong, is there a better way to migrate or update the timestamps so they appear in same orders as they were initially published?  Kindly let me know.  Thanks once again for everything!! Lots of blessings.

**Comment by albertobeta** [comment]
Hello! The order of episodes is based on the file timestamp, by migrating you lost the original timestamps.
If you do not want to change them manually one by one via the PG admin area, you could create a script that reads all the files in the media folder, and that for each of them reads the date from the filename (PG always prepends the date to the file name) and "touches" the file thus modifying the timestamp and restoring the original order. If you don't have coding skills, maybe it will take less time to change dates one by one. cheers

**Comment by pbassham** [comment]
I made a bash script to do one way of fixing this.

1. Name all the files in alphabetical order and REMOVE SPACES from the file names
2. Run this script to update the timestamps:
`#!/usr/bin/env bash 
for file in `find /path/to/folder/media -type f -name '*.mp3' | sort`; do touch "$file"; sleep 2.1; done`

I had to put a 2 second delay in there. Anything smaller wasnt sufficient, but this makes it take a while to run if you have a lot of files. There is probably a faster way, but this works.
